<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TextEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="autoFormatting">
       <set>QTextEdit::AutoAll</set>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOPen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionFind"/>
    <addaction name="actionFind_next"/>
   </widget>
   <widget class="QMenu" name="menuFont">
    <property name="title">
     <string>Font</string>
    </property>
    <addaction name="actionChange_font"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom"/>
    <addaction name="actionZoom_0"/>
    <addaction name="separator"/>
    <addaction name="actionStatus_bar"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout_software"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFont"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNew_window">
   <property name="text">
    <string>New window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionOPen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFind_next">
   <property name="text">
    <string>Find next</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="text">
    <string>Select all</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="text">
    <string>Zoom +</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_0">
   <property name="text">
    <string>Zoom -</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionStatus_bar">
   <property name="text">
    <string>Status bar</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+H</string>
   </property>
  </action>
  <action name="actionAbout_software">
   <property name="text">
    <string>About software</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionChange_font">
   <property name="text">
    <string>Change font</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
